assets/js/modules/analytics/datastore/accounts.js

```diff
@@ -137,6 +138,13 @@ const baseActions = {
 		).invalidateResolutionForStoreSelector( 'getAccounts' );
 	},
 
+	resetPropertyAndProfileIDs() {
+		return {
+			payload: {},
+			type: RESET_PROPERTY_AND_PROFILE,
+		};
+	},
```

Comment:
It's a bit odd to me the names and constants are so close but don't quite match. Let's either rename this `resetPropertyAndProfileIDs`, or rename the action type constant `RESET_PROPERTY_AND_PROFILE_IDS` 🙂

Reply:
Renamed the constant to `RESET_PROPERTY_AND_PROFILE_IDS`